warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexListIteratorJexl.java

```diff
@@ -100,14 +100,27 @@ public SortedKeyValueIterator<Key,Value> deepCopy(IteratorEnvironment env) {
         return new DatawaveFieldIndexListIteratorJexl(this, env);
     }
 
+    @Override
+    public String toStringNoQueryId() {
```

Comment:
Why override this method if it is identical to the parent?

Reply:
Fixed